DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Model/EncryptFunctionTypes.cs

```diff
@@ -0,0 +1,13 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Model
+{
+    public enum EncryptFunctionTypes
```

Comment:
usually we name enum as single type form, could be EncryptFunctionType or EncryptMethodType?